web-store/src/main/resources/static/Rest.txt

```diff
@@ -0,0 +1,189 @@
+Models
+
+User {
+  "id": String,
+  "username": String,
+  "firstName": String,
+  "lastName": String,
+  "email": String,
+  "password": String,
+  "phone": NUMBER,
+}
+
+Item { 
+  "id": String,
+  "title": String,
+  "cost": Integer,
+  "weight": Double,
+  "manufacturer": String
+}
+
+
+
+requset - create a new user
+POST api/v1/users HTTP/1.1
+Host: https://weldingstoreexample.com
+Content-Type:  application/json
+{
+  "id": "1",
+  "username": "Ivan1",
+  "firstName": "Ivan",
+  "lastName": "Ivanov",
+  "email": "ivanov@g.com",
+  "password": "123",
+  "phone": 543,
+}
+
+response:
+{
+  "code": 200,
+  "message": "ok"
+}

```

Comment:
Why do you have rest response in json format?
it should be

Response:
Status: 200 OK
Body: <empty>